homework/insensitive-palindrom/insensitivePalindrom.cpp

```diff
@@ -0,0 +1,18 @@
+#include <string>
+#include <algorithm>
+#include "insensitivePalindrom.hpp"
+
+bool is_palindrome(std::string testString) {
+    
+    //std::string testString;
+    testString.erase(std::remove_if(testString.begin(), testString.end(), isspace), testString.end());
+    testString.erase(std::remove_if(testString.begin(),
+                                    testString.end(),
+                                    [](auto const & c) -> bool {
+                                        return !std::isalnum(c);
+                                    }),
+                                    testString.end()
+                                    );
+    std::transform(testString.begin(), testString.end(), testString.begin(), [](auto & c){return std::tolower(c);});
```

Comment:
Use transform and erse -> iterator returned by transform will be last valid, and rest of it you can erase
So you avoid this two std::remove_if